can-globals-proto.js

```diff
@@ -7,6 +7,9 @@ function dispatch(key, value) {
 	var handlers = this.eventHandlers[key];
 	if (handlers) {
 		var handlersCopy = handlers.slice();
+		if (typeof value === 'function') {
+			value = value();
```

Comment:
Will this fix my can-util problem? Looks like it'll dispatch undefined, when what I need is the default value.

Reply:
Nevermind, this should fix my problem, but if value() returns undefined I still expect it to return the default value.